README.md

```diff
@@ -40,6 +37,36 @@ Once a process terminates, the streams will become closed but not unset.
 * `$stdout`
 * `$stderr`
 
+Each of these implement the underlying
+[`DuplexStreamInterface`](https://github.com/reactphp/stream#duplexstreaminterface)
+and you can use any of its events and methods as usual:
+
+```php
+$process->stdout->on('data', function ($chunk) {
```

Comment:
I'd rather leave this like this in order to emphasize this is not necessarily a complete "data message", but could possibly be only a small "data chunk". This is also in line with all other stream documentations.

Any objections / recommendations?